### PR TITLE
Implement configuration token with code restructure

### PR DESCRIPTION
**Pull request by TarasMaliuchenko**

### CODE REVIEW

File: code/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Authlib==0.14.3
```
Comment (AnastasiiaRozlyvan) [inline code/requirements.txt:1]: I think Authlib is an extra dependency, since we switched from Authlib to pyjwt

File: LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+
+The MIT License (MIT)
+
+Copyright (c) 2020 Michael Auger
```
Comment (AnastasiiaRozlyvan) [inline LICENSE.md:4]: Should be **Copyright (c) 2021 Cisco SecureX**

File: code/requirements.txt (hunk 1 above)
Comment (TarasMaliuchenko) [reply code/requirements.txt]: Fixed

File: LICENSE.md (hunk 2 above)
Comment (TarasMaliuchenko) [reply LICENSE.md]: Fixed

File: README.md [hunk 3]
```diff
@@ -1,227 +1,77 @@
...
-**NOTE**. If you are experiencing any problems with running the command then
-check the [AWS Common Errors](aws/CommonErrors.md) guide on troubleshooting
-of some most common types of errors.
+ 2. Once the container is built, and an image is successfully created, start your container using the `docker run` command and specify the name of the image we have just created.  By default, the container will listen for HTTP requests using port 9090.
 
-Once the Lambda has been deployed, make sure to save the public `URL` to your
-Lambda returned by Zappa. It will look like this:
 ```
-https://<RANDOM_ID>.execute-api.<AWS_REGION>.amazonaws.com/<STAGE>
+docker run -dp 9090:9090 --name tr-05-serverless-urlscanio tr-05-serverless-urlscanio
```
Comment (AnastasiiaRozlyvan) [inline README.md:60]: It's just a naming and there is nothing critical but for I'm not sure if the usage of the word `serverless` is correct here. Perhaps if Ien used the name `tr-05-ibm-xforce-exchange` for X-Force then it is also worth doing `tr-05-urlscanio` here just for consistency.

File: build.sh [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+echo
+
+module_name="Urlscanio Relay"
+image_name="tr-05-serverless-urlscanio"
+
+CONFIG_FILE=code/container_settings.json
+if [ -f $CONFIG_FILE ]; then
+   echo
+   echo "The configuration file (container_settings.json) already exists."
+   echo "Remove this file if you wish to regenerate a secret_key."
```
Comment (AnastasiiaRozlyvan) [inline build.sh:16]: And here there is an unnecessary mention of `secret_key`. I don't think we need it anymore.

File: README.md (hunk 3 above)
Comment (TarasMaliuchenko) [reply README.md]: Fixed

File: scripts/start.sh [hunk 5]
```diff
@@ -0,0 +1,14 @@
+#! /usr/bin/env sh
+if [[ -z "${DEBUG}" ]]; then
+	echo "[start.sh] DEBUG MODE OFF" >> /var/log/messages
+else
+	echo "[start.sh] DEBUG MODE ON" >> /var/log/messages
+	echo "[start.sh] ............." >> /var/log/messages
+	echo "[start.sh] Integration Module: " `jq -r .NAME /app/container_settings.json` >> /var/log/messages
+	echo "[start.sh]            Version: " `jq -r .VERSION /app/container_settings.json` >> /var/log/messages
+	echo "[start.sh]         Secret Key: " `jq -r .SECRET_KEY /app/container_settings.json` >> /var/log/messages
```
Comment (AnastasiiaRozlyvan) [inline scripts/start.sh:9]: There is also an extra `Secret Key`

**Review by AnastasiiaRozlyvan — APPROVED** [review]

**Pull request merged**